Osushi/View/PostList/PostRowView.swift

```diff
@@ -7,10 +7,8 @@ struct PostRowView: View {
     
     var heading1Text: String {
         let lines = markdownContent.components(separatedBy: "\n")
-        for line in lines {
-            if line.starts(with: "# ") {
-                return line.replacingOccurrences(of: "# ", with: "").trimmingCharacters(in: .whitespaces)
-            }
+        for line in lines where  line.starts(with: "# ") {
```

Comment:
[nits]
半角スペースが2つ続いちゃってる

```suggestion
        for line in lines where line.starts(with: "# ") {
```